projects/ardin_mb/model_ardin.cc

```diff
@@ -57,7 +57,7 @@ void modelDefinition(NNmodel &model)
 {
     using namespace BoBRobotics;
     model.setDT(MBParamsArdin::timestepMs);
-    model.setName("mb_memory_ardin");
+    model.setName("ardin_mb");
```

Comment:
this definitely breaks the runtime dynamic linking as it looks to load a model called mb_memory_ardin...